membership-attribute-service/app/sources/SalesforceCSVExport.scala

```diff
@@ -1,26 +0,0 @@
-package sources
```

Comment:
I should have removed the SalesforceCSVExport stuff in https://github.com/guardian/members-data-api/pull/283

But I'm doing it here because otherwise I'd be maintaining its reference to Attributes

Reply:
🔪